lib/Doctrine/ORM/Query/Filter/SQLFilter.php

```diff
@@ -112,6 +112,11 @@ abstract class SQLFilter
         return serialize($this->parameters);
     }
 
+    final protected function getEntityManager(): EntityManagerInterface
+    {
+        return $this->em;
+    }
+    
```

Comment:
```suggestion

```

Reply:
I didn't look to see what the standard is for Doctrine here, but having an additional empty line at the end of a file is recommended for portability reasons.  Unix/POSIX systems generally assume a trailing empty line as files are concatenated together.  Without the final line, the last line will be merged with the first line of the next file, sometimes causing syntax errors.

Reply:
This is not the end of the file, is it? There should definitely be a blank line between methods though.

Reply:
Sorry, I thought it was.  I have no idea what this is then.  This snippet is one line.  I see no way to view the full file with requested changes here.  I'm not very familiar with this interface and am finding it to be incredibly frustrating to work with at this depth.

Feel free to commit it.  I thought I added a line between the method.  At least when I click to "view file", that's what I see.

Reply:
I think there is a line between the methods, so all is well for me. I was saying that for @SenseException , not for you.

Reply:
It's not about the line but empty spaces. "Whitespace found at end of line" is more that the whole line consists of empty spaces.

Reply:
I think they must have been removed cause I don't see the spaces in the diff when trying to highlight them with my cursor.

Reply:
I see that it is still mentioned by phpcs in the current build result: https://travis-ci.org/doctrine/orm/jobs/641467131#L562. Does the IDE not show them?

Reply:
vim does show them, not sure why Github doesn't. I just force pushed.